includes/admin/class.llms.admin.notices.php

```diff
@@ -300,14 +301,13 @@ public static function output_notice( $notice_id ) {
 
 			$notice = self::get_notice( $notice_id );
 
-			if ( empty( $notice ) ) {
-				return;
-			}
-
 			// Don't output those rogue empty notices I can't find.
 			// @todo find the source.
-			if ( empty( $notice['template'] ) && empty( $notice['html'] ) ) {
+			if ( empty( $notice ) || ( empty( $notice['template'] ) && empty( $notice['html'] ) ) ) {
 				self::delete_notice( $notice_id );
+
+				/** @noinspection PhpUnnecessaryStopStatementInspection */
```

Comment:
What's this? an IDE code sniffer ignore or something?

I'd prefer to not start adding IDE-specific attributes into the codebase. My thinking here is that if we start to add IDE-specific attributes for *you* we also need to consider that we might need to start adding IDE specific attributes for other developers. This doesn't *affect* anything greatly but imagine a team with 10 developers all using different IDEs (I know there maybe isn't that many robust / popular IDEs for PHP, but still). I can imagine that over time the codebase could become pretty "ugly"

Is there another way to satisfy whatever is causing your IDE to throw a warning / error / whatever here? Can the code be rewritten as to not throw this kind of warning / error / etc...?

Reply:
PhpStorm incorrectly issues a "`return` is unnecessary as the last statement in a method" warning. Apparently, the inspection does check if HTML can be output in the method.

I'm OK taking this inspection suppressor out because it's wrong and only a minor nuisance.

There are probably other IDE comment helpers that I might feel more strongly about, but not this one.